config/packages/app_event_admin.yaml

```diff
@@ -4,3 +4,38 @@ sulu_admin:
             routes:
                 list: 'app.get_event_list'
                 detail: 'app.get_event'
+    field_type_options:
+        selection:
+            event_selection:
+                default_type: 'list_overlay'
+                resource_key: 'events'
+                view:
+                    name: 'app.event.edit_form'
+                    result_to_view:
+                        id: 'id'
+                types:
+                    list_overlay:
+                        adapter: 'table'
+                        list_key: 'events'
```

Comment:
This reference the already existing `events.xml`: here: https://github.com/sulu/sulu-demo/blob/example/custom-entity/config/lists/events.xml